tests/unit/Storefront/Controller/ProductControllerTest.php

```diff
@@ -185,29 +179,10 @@ public function testQuickViewMinimal(): void
         static::assertInstanceOf(ProductQuickViewWidgetLoadedHook::class, $this->controller->calledHook);
     }
 
-    public function testSaveReviewDeactivated(): void
```

Comment:
should we leave this test? :thinking:

Reply:
As in this case the `ProductReviewLoader` (and therefore also the route), is only mocked, it would make no effect, I therefore added the unit tests for the route itself. Maybe for the integration test (if there exists one) it could be tested.

Reply:
ah yes, sure :+1: 

there is indeed an integration test: https://github.com/shopware/shopware/blob/trunk/tests/integration/Storefront/Controller/ProductControllerTest.php